src/applications/vaos/components/AppointmentRequestListItem.jsx

```diff
@@ -82,7 +83,7 @@ export default class AppointmentRequestListItem extends React.Component {
           {!showCancelButton || canceled ? null : (
             <div>
               <button
-                className="usa-button-secondary vads-u-margin--0 vads-u-flex--0"
+                className="vaos-appts__cancel-btn usa-button-secondary vads-u-margin--0 vads-u-flex--0"
```

Comment:
Is this meant to be double underscore?

Reply:
It is, that's part of the BEM convention that we sort of follow for CSS classes.

Reply:
It is, that's part of the BEM convention that we sort of follow for CSS classes.